lib/bot.js

```diff
@@ -251,6 +251,11 @@ class Bot {
       return;
     }
 
+    if (message.bot_id == 'B42KVL3PF') { // CHANGEME
```

Comment:
This bot ID should be stored in/loaded from the config.json file, and documentation added to the README. It may even be worth making it an array of bot IDs to ignore.